### PR TITLE
(fix) Use state uuid as opposed to concept uuid in program state inte…

### PR DESCRIPTION
**Pull request by CynthiaKamau**
…ractive view

## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- Use state uuid as opposed to concept uuid in program state questions in interactive view

## Screenshots

Screenshot 2024-07-09 at 11 40 29

### CODE REVIEW

File: src/components/interactive-builder/add-question.modal.tsx [hunk 1]
```diff
@@ -238,7 +238,7 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
             selectedConcept?.allowDecimal === false && { disallowDecimals: true }),
           ...(questionType === 'programState' && {
             answers: selectedProgramState.map((answer) => ({
-              value: answer.concept.uuid,
+              value: answer.uuid,
```
Comment (arodidev) [inline src/components/interactive-builder/add-question.modal.tsx:241]: Should the `answer` object align with [this answer interface](https://github.com/openmrs/openmrs-esm-form-builder/blob/8426207598bc6cab16a84279839a2f1d28ac20f9/src/types.ts#L150), or should the interface be updated to reflect some of the additions/omissions?

**Review by arodidev — COMMENTED** [review]
Thanks @CynthiaKamau, left a review comment.

File: src/components/interactive-builder/add-question.modal.tsx (hunk 1 above)
Comment (CynthiaKamau) [reply src/components/interactive-builder/add-question.modal.tsx:241]: The value for this specific question is the state uuid and not the concept uuid, i think its more of [this](https://github.com/openmrs/openmrs-esm-form-builder/blob/8426207598bc6cab16a84279839a2f1d28ac20f9/src/types.ts#L166C18-L166C31)

Comment (CynthiaKamau) [reply src/components/interactive-builder/add-question.modal.tsx:241]: Not quite, the type `ProgramState` contains a concept object and a uuid

**Review by arodidev — APPROVED** [review]
Thanks @CynthiaKamau

**Pull request merged**